lucille-core/src/test/java/com/kmwllc/lucille/message/KafkaMessengerTest.java

```
@@ -0,0 +1,168 @@
+    }
+  }
+
+  @Test
+  public void testSendEvent2() throws Exception {
+    Config config = ConfigFactory.load("WorkerPoolTest/config.conf");
+    Config config2 = ConfigFactory.load("WorkerTest/config.conf");
+    Document doc = Document.create("doc1");
+    try (MockedStatic<KafkaUtils> kafkaUtils = Mockito.mockStatic(KafkaUtils.class);
+        MockedStatic<RandomStringUtils> randomUtils = Mockito.mockStatic(RandomStringUtils.class)) {
+      Future<RecordMetadata> mockResult = Mockito.mock(Future.class);
+      KafkaProducer<String, String> mockProducer = (KafkaProducer<String, String>) Mockito.mock(KafkaProducer.class);
+      Mockito.when(mockProducer.send(Mockito.any())).thenReturn(mockResult);
+      KafkaConsumer<String, KafkaDocument> mockConsumer = (KafkaConsumer<String, KafkaDocument>) Mockito.mock(KafkaConsumer.class);
+      randomUtils.when(() -> RandomStringUtils.randomAlphanumeric(8)).thenReturn("random");
+      kafkaUtils.when(() -> {
+        KafkaUtils.createEventProducer(config);
+      }).thenReturn(null);
+      kafkaUtils.when(() -> {
+        KafkaUtils.createEventProducer(config2);
+      }).thenReturn(mockProducer);
+      kafkaUtils.when(() -> {
+        KafkaUtils.createDocumentConsumer(config2, "com.kmwllc.lucille-worker-foo2-random");
+      }).thenReturn(mockConsumer);
+      kafkaUtils.when(() -> {
+        KafkaUtils.createDocumentConsumer(config, "com.kmwllc.lucille-worker-foo-random");
+      }).thenReturn(mockConsumer);
+      kafkaUtils.when(() -> {
+        KafkaUtils.getEventTopicName("foo2", doc.getRunId());
+      }).thenReturn("foo2_event_id");
+      KafkaWorkerMessenger messenger = new KafkaWorkerMessenger(config, "foo");
+      KafkaWorkerMessenger messenger2 = new KafkaWorkerMessenger(config2, "foo2");;
+      try {
+        messenger.sendEvent(doc, "message", Event.Type.FAIL);
+        assertTrue(true);
+      } catch (Exception e) {
+        // we want this to not throw an expection
+        assertTrue(false);
```

same comment as elsewhere -- don't need the try/catch, just let any exceptions bubble up and fail the test